tests/unit/jobserver/test_forms.py

```diff
@@ -104,27 +104,42 @@ def test_jobrequestcreateform_with_bad_codelists(
     ],
 )
 def test_workspacecreateform_success(name, cleaned_name):
+    """
+    When the form is instantiated with valid data then:
+        * The form validates.
+        * The data ends up in the cleaned_data.
+        * The name field is cleaned by converting to lower case.
+    """
```

Comment:
I'm reluctant to add a comment (docstring) that's an English-language description of the implementation. If we thought of the implementation as a black-box, then what could we say about the intention of this function?

I think the intention of this function is to test that form and field validation works correctly (where validation encompasses cleaning). Perhaps renaming the function `test_workspacecreateform_validation` would help communicate the intention?

Reply:
I think one good alternative might be making the test more atomic -- this test now tests multiple things, I think. I haven't yet thought about the detail of how to do that.